### ISSUE TITLE
MediaManagerImplementation does not contains MediaRemoteControl

### ISSUE THREAD

**Issue by MrSerWin** [martijn00/XamarinMediaManager#442]
## 🐛 Bug Report
MediaManagerImplementation does not contains MediaRemoteControl

### Configuration
I've implemented MediaRemoteControl renderer for iOS on v. 0.4.x and it worked, but after update to 0.5.3 I have an error 
> MediaManagerImplementation does not contains MediaRemoteControl
[documentation](https://github.com/martijn00/XamarinMediaManager#important) 

my old code 
```
[Register(Application.AppName)]
    //[Register("AppDelegate")]
    public class MediaFormsApplication : UIApplication
    {
        private MediaManagerImplementation MediaManager => (MediaManagerImplementation)CrossMediaManager.Current;

        public override void RemoteControlReceived(UIEvent uiEvent)
        {
            // MediaManagerImplementation.MediaRemoteControl.RemoteControlReceived(UIEvent)
            MediaManager.MediaRemoteControl.RemoteControlReceived(uiEvent);
        }

    }
```
**Version:** 0.5.3

**Platform:** 
- [x] :iphone: iOS
- [ ] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by karak** [comment]
I've got the same error to revert upgrading.

**Comment by cklenk** [comment]
I noticed the same thing today too!

**Comment by martijn00** [comment]
This is fixed in 0.5.9. There is no need to add RemoteControlReceived anymore.
